src/main/java/org/ow2/proactive/connector/iaas/rest/InstanceRest.java

```diff
@@ -63,8 +63,8 @@
     @Produces("application/json")
     @Path("{infrastructureId}/instances")
     public Response createInstance(@PathParam("infrastructureId") String infrastructureId, final String instanceJson) {
-        log.info("Received create request for infrastructure " + infrastructureId + " with parameters " + instanceJson);
         Instance instance = JacksonUtil.convertFromJson(instanceJson, Instance.class);
+        log.info("Received create request for infrastructure " + infrastructureId + " with parameters " + instance);
```

Comment:
"for infrastructure " => "for infrastructure ID " maybe is better but up to you

Reply:
Leaving in small cap, just as other id in the logs